docs/src/docs/asciidoc/index.adoc

```diff
@@ -34,28 +37,30 @@ Spring Security OAuth2 Boot simplifies standing up an OAuth 2.0 Authorization Se
 
 You need to stand up your own authorization server if:
 
-- You want to delegate the operations of sign-in, sign-out, and password recovery to a separate service (also called _identity federation_) that you want to manage yourself, and
-- You want to use the OAuth 2.0 protocol for this separate service to coordinate with other services
+* You want to delegate the operations of sign-in, sign-out, and password recovery to a separate service (also called _identity federation_) that you want to manage yourself and
+* You want to use the OAuth 2.0 protocol for this separate service to coordinate with other services
 
 === Dependencies
 
-To leverage the auto-configuration features in this library, you'll need `spring-security-oauth2`, which has the OAuth 2.0 primitives and `spring-security-oauth2-autoconfigure`.
-Note that you'll need to specify the version for `spring-security-oauth2-autoconfigure` since it isn't managed by Spring Boot any longer, though, it should match Boot's version anyway.
+To use the auto-configuration features in this library, you need `spring-security-oauth2`, which has the OAuth 2.0 primitives and `spring-security-oauth2-autoconfigure`.
+Note that you need to specify the version for `spring-security-oauth2-autoconfigure`. since it is not managed by Spring Boot any longer, though it should match Boot's version anyway.
```

Comment:
Is that supposed to be a comma? "...the version for `spring-security-oauth2-autoconfigure`**,** since it is..."

Reply:
Yup. Thanks for catching it. :)